internal/generator/ecc/template/point.go.tmpl

```diff
@@ -796,9 +840,15 @@ func (p *{{$TJacobian}}) ClearCofactor(a *{{$TJacobian}}) *{{$TJacobian}} {
 	L1.AddAssign(&tmp)
 
 	p.phi(&L1).AddAssign(&L0)
+{{ else}}
+    var c1 big.Int
```

Comment:
why do we want to allow for a non-GLV path for these curves?

Reply:
1- there is a boolean flag that results in errors in the code when set to false. I believe that the first intention was to find a way around the GLV patent (now expired).
2- this should be cleaner way to have other curves without GLV (e.g. STARK curve, NIST P-256..)

Reply:
ok -- I'ld have taken a shortcut and just panicked for curve we know we have GLV, instead of writing the non-glv path but.. sure :)